google-cloud-storage/src/test/java/com/google/cloud/storage/BlobTest.java

```diff
@@ -129,7 +129,11 @@
           .setRetentionExpirationTime(RETENTION_EXPIRATION_TIME)
           .build();
   private static final BlobInfo BLOB_INFO =
-      BlobInfo.newBuilder("b", "n").setMetageneration(42L).build();
+      BlobInfo.newBuilder("b", "n", 12345678L).setMetageneration(42L).build();
```

Comment:
what `"b"` and `"n"` indicates here?

Reply:
"b" for bucket, "n" for name. Please not, those names were before me. I just added generation value.